src/scales/scale.time.js

```diff
@@ -548,8 +548,8 @@ module.exports = function(Chart) {
 
 			if (timestamps.length) {
 				timestamps = arrayUnique(timestamps).sort(sorter);
-				min = Math.min(min, timestamps[0]);
-				max = Math.max(max, timestamps[timestamps.length - 1]);
+				min = min === MAX_INTEGER ? timestamps[0] : min;
```

Comment:
I think that's wrong, if `min !== MAX_INTEGER`, we still want `Math.min(min, timestamps[0]);` because `min` can change line 545 to a lower value. I don't understand why these 2 lines need to be modified, @Andgoncharov can you elaborate why these changes?

Reply:
I agree about elaborating this. @Andgoncharov could you post images with and without these changes? I also agree that we need to keep the `Math.min()` call.

Reply:
Hi @Simonbrunel, @Etimberg,

I've created a fiddle https://jsfiddle.net/andgoncharov/8bgen7b7/ to display the problem. When I change options.time.min and options.time.max values, I expect that chart is displayed within specified borders and ticks are also distributed between those values. Currently, min and max values ain't taken into account if they are inside (timestamps[0], timestamps[length - 1]) interval.

When I apply my changes, the updated chart contains a number of x-axis labels according to the settings I've specified upon creation of the chart. Or may be I misunderstood something?